Skeletorfile.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use NiftyCo\Skeletor\Skeletor;
+
+return function (Skeletor $skeletor) {
+    $skeletor->intro('Starting Fission setup...');
+
+
+    if (!$skeletor->exists('.env')) {
+        $env = $skeletor->readFile('.env.example');
+        $skeletor->writeFile('.env', $env);
+        $skeletor->info('✓ .env file created.');
+    }
+
+    $skeletor->spin('Generating application key...', function () use ($skeletor) {
+        $skeletor->exec(['php', 'artisan', 'key:generate', '--ansi']);
+    });
+    $skeletor->info('✓ Application key generated.');
+
+    if (!$skeletor->exists('node_modules')) {
+        $skeletor->spin('Running npm install...', function () use ($skeletor) {
+            $skeletor->exec(['npm', 'install']);
+        });
+        $skeletor->info('✓ npm dependencies installed.');
+    }
+
+    if ($skeletor->exists($authPath = $_SERVER['HOME'] . '/Code/flux-auth.json')) {
+        $skeletor->spin('Copying auth.json from ~/Code', function () use ($skeletor, $authPath) {
+            $authJson = $skeletor->readFile($authPath);
+            $skeletor->writeFile('auth.json', $authJson);
+        });
+        $skeletor->info('✓ auth.json copied.');
+
+        $skeletor->spin('Activating Flux Pro', function () use ($skeletor) {
+            $skeletor->exec(['composer', 'install']);
```

Comment:
When running our first `composer install` either by executing the `create-project` command or `install` ourselves, we get prompted for flux pro auth, so would it make more sense to not have it in the `composer.json` by default and in this step we do the composer command to set it, and prompt for the auth ourselves that way it's all done in one fell swoop?